CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+- **Breaking**: change ``cartridge pack`` naming strategy:
+  * RPM: `<app-name>-<version>[.<suffix>]-1.<arch>.rpm`,
+  * DEB: `<app-name>-<version>[.<suffix>]-1_<arch>.deb`,
+  * TGZ: `<app-name>-<version>[.<suffix>].tar.gz`.
+  `<version>` is `git describe` output normalized to `X.Y.Z.N` or ``--version`` value.
+- **Breaking**: ``cartridge pack --version`` value is used as provided without normalization.
+- **Breaking**: ``cartridge pack`` ``--version`` and ``--suffix`` is validated
+  after RPM/DEB conventions for corresponding type (see
+  https://www.debian.org/doc/manuals/debian-reference/ch02.en.html#_debian_package_file_names
+  and http://ftp.rpm.org/max-rpm/ch-rpm-file-format.html).
+  For example, dashes in rpm version (`1.2.3-0`) is no longer supported.
+- RPM is now built for `noarch` instead of `x86_64`.
```

Comment:
Maybe keep it x86_64 for now and implement achitecture detection later? At least we can have compiled modules, which will not run on another architecture. Also we pack tarantool enterprise executable into the application if a user works under tarantool enterprise.

Reply:
Restored old behavior